onnxruntime/core/providers/shared/initializer_view/initializer_view.cc

```diff
@@ -0,0 +1,20 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/shared/initializer_view/initializer_view.h"
+#include <utility>
+
+namespace onnxruntime {
+common::Status InitializerView::Create(
+    const ONNX_NAMESPACE::TensorProto& tensor_proto, std::optional<InitializerView>& initializer) {
```

Comment:
> std::optional<InitializerView>& initializer)  [](http://example.com/codeflow?start=52&length=46)

Why not simply instantiate it on stack?

Reply:
Getting rid of Optional and istantiated it on stack